.changeset/early-spiders-hear.md

```diff
@@ -0,0 +1,5 @@
+---
+"victory-native": minor
```

Comment:
PR conversation has "Breaking Change" checked. this PR isn't actually breaking is it?

Reply:
Sorry. Initially, there was a breaking change as I was removing the grid property, but I added that back and did not update the descrtiption. Removed that check mark